### PR TITLE
small spelling and grammar fixups to news285

### PR DESCRIPTION
**Pull request by LarryRuane**
Follow-up to #1468, I was unavailabale to review pre-merge

### CODE REVIEW

**Review by bitschmidty — COMMENTED** [review]
Noticing a missing period after the lede as well. Will defer to @harding on these other suggestions/updates.

**Comment by LarryRuane** [comment]
Good catch, Mike, I force-pushed that fix (add the missing period).

**Comment by LarryRuane** [comment]
Good catch, Mike, I force-pushed that fix (add the missing period).

**Comment by LarryRuane** [comment]
Force-pushed to resolve merge conflicts with #1476 (podcast timestamps).

File: _posts/en/newsletters/2024-01-17-newsletter.md [hunk 1]
```diff
@@ -262,8 +262,8 @@ Proposals (BIPs)][bips repo], [Lightning BOLTs][bolts repo], and
   any onchain fees directly <!-- endogenously --> included in the
   commitment transaction and the presigned HTLC-Success and HTLC-Timeout
   transactions (HTLC-X transactions).  If the funding party doesn't have
-  much funds in the channel and if feerates rise, the funding party may
-  not be able accept a new incoming payment because they don't have
+  many funds in the channel and if feerates rise, the funding party may
```
Comment (murchandamus) [inline _posts/en/newsletters/2024-01-17-newsletter.md:265]: "Much" seems right to me. Isn’t _funds_ an uncountable noun? Either way, perhaps:

```suggestion
  transactions (HTLC-X transactions).  If the funding party is low on
  funds in the channel and if feerates rise, the funding party may
```

**Review by murchandamus — COMMENTED** [review]
ACK except comment on funds

**Comment by LarryRuane** [comment]
Force-pushed Murch's suggestion, I like that much better than what I suggested, thanks!

**Review by bitschmidty — APPROVED** [review]

**Pull request merged**